translations/messages.fr.yaml

```diff
@@ -1 +1,24 @@
+Contribute: "Contribuez au projet\xA0:"
```

Comment:
you should use lower & snake case

Reply:
or camelCame, but without upper case on first letter

Reply:
I don't think we need to add `\xA0` : if we need to have non-breaking spaces, we should use css instead

Reply:
In projects that im working keys are camelCase with kind of namespaces to start the string example : 
user.firstName
project.shortTitle
project.title

Thats very convenient when you are looking for existing keys imo

Reply:
sure, let's do it! But as you said, the first letter is always a lowercase one